cmd/earthly/main.go

```diff
@@ -1278,6 +1280,43 @@ func (app *earthlyApp) processDeprecatedCommandOptions(context *cli.Context, cfg
 	return nil
 }
 
+func (app *earthlyApp) unhideFlags(ctx context.Context) error {
+	var err error
+	if os.Getenv("EARTHLY_AUTOCOMPLETE_HIDDEN") != "" && os.Getenv("COMP_POINT") == "" { // TODO delete this check after 2021-03-01
```

Comment:
randomly picked date; this isn't an advertised feature -- no need to support long term.

Reply:
That seems much longer term than needed. It's only really a developer-feature, I don't think it's something that we need to worry about. I suspect I could just have skipped this warning all together and nobody would have noticed.

Reply:
yeah fair -- 0.7 was just an example, saying "after v0.6" could work too